MmSupervisorPkg/Core/PrivilegeMgmt/PrivilegeMgmt.h

```diff
@@ -118,6 +106,18 @@ SyncMmEntryContextToCpl3 (
   VOID
   );
 
+/**
+  Invoke specified routine on specified core in CPL 3.
+**/

```

Comment:
For this function in particular, I'd provide argument descriptions since manually providing a variable argument count is a bit unusual (but understandable in this case).

Reply:
Thanks, updated.